### DISCUSSION TITLE
Monet theming is not there.

### DISCUSSION THREAD

**Discussion by chathura-de-silva** [Docile-Alligator/Infinity-For-Reddit#1089] [Q&A] [answered]
I am on Android 12 on top of One UI 4.1. Though many people have mentioned that infinity supports material u theming, for me it just stays in a one default colour and even in settings there are bunch of colour palletes to choose from and thats it. Why this is happening? If this is something from the developers end, please make it work on Android 12.

**Comment by Wladefant** [comment, marked as answer]
Hello @chathura7357 ,
It seems to be a bug: https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/1017
